_headers

```diff
@@ -1,2 +1,5 @@
 /*
   Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
+  Content-Security-Policy: object-src 'none'; frame-ancestors 'none';
```

Comment:
I see you, too, decided that a more restrictive CSP wasn't something you wanted to mess with :)